yacc/console/gram.y

```diff
@@ -493,16 +497,28 @@ distribution_key_entry:
 		}
 	}
 
-relation_attach_stmt:
-	ATTACH RELATION any_id DISTRIBUTION KEY distribution_key_argument_list
+distributed_relation_def:
+	RELATION any_id DISTRIBUTION KEY distribution_key_argument_list
 	{
 		$$ = &DistributedRelation{
-			Name: 	 $3,
-			DistributionKey: $6,
+			Name: 	 $2,
+			DistributionKey: $5,
 		}
 	}
 
 
+distributed_relation_list_def:
+	distributed_relation_def {
+		$$ = []*DistributedRelation{$1}
+	} | distributed_relation_list_def distributed_relation_def {
```

Comment:
```suggestion
	} | distributed_relation_list_def TCOMMA distributed_relation_def {
```
Might use brackets as well

Reply:
we already have comma in DISTRIBUTION KEY clause. Relations are separated with RELATION token. 

Like this
```

		ALTER DISTRIBUTION 
			ds1 
		ATTACH
			RELATION t DISTRIBUTION KEY id1, id2 HASH FUNCTION murmur
			RELATION t2 DISTRIBUTION KEY xd1, xd2 HASH FUNCTION city
```